tests/test_convert.py

```diff
@@ -10,10 +10,14 @@
     Clbit,
     Qubit,
 )
+from qiskit.transpiler import Layout
+from qiskit.transpiler.layout import TranspileLayout
 from qiskit.quantum_info import Operator
 
 from qiskit_qasm3_import import parse, ConversionError
 
+# pylint: disable=too-many-lines
+
```

Comment:
Let's disable this project-wide in `pyproject.toml` - it's a stupid rule anyway.